css/about.css

```diff
@@ -0,0 +1,127 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
```

Comment:
You can take the normalize or reset CSS and also put it in its own file and reference it on each of your different pages.